test/cases/active_job_serializer_test.rb

```diff
@@ -0,0 +1,47 @@
+require 'abstract_unit'
+
+require 'fixtures/project'
+require 'fixtures/person'
+require 'active_job_object_serializer'
+require 'active_resource/active_job_serializer'
+
+class ActiveJobSerializerTest < ActiveSupport::TestCase
+
+  def setup
+    @klass = ActiveResource::ActiveJobSerializer
+    @person = Person.new({ id: 2, name: 'David' }).tap do |person|
+      person.persisted = true
+    end
+    @person_json = jsonify(@person)
+    @project = Project.new({ id: 1, name: 'Ruby on Rails' }).tap do |project|
+      project.prefix_options[:person_id] = @person.id
+    end
+    @project_json = jsonify(@project)
```

Comment:
Think the `@person_json` and `@project_json` ivars obscure the tests too much. I think we should inline them. And I'd also like to see the `jsonify` helper removed, so the hash isn't necessarily dynamically generated.